features/tracker-allowlist.json

```
@@ -515,73 +631,95 @@
             "bigcrunch.com": {
                 "rules": [
                     {
-                        "rule": "ship.bigcrunch.com/prebid",
-                        "domains": ["tvtropes.org"],
-                        "reason": ["https://github.com/duckduckgo/privacy-configuration/pull/4006"]
-                    },
-                    {
-                        "rule": "lh.bigcrunch.com/main.js",
-                        "domains": ["snopes.com"],
-                        "reason": ["https://github.com/duckduckgo/privacy-configuration/pull/4007"]
+                        "rule": "ship.bigcrunch.com/ads.js?asdf=xxx",
```

```suggestion
                        "rule": "ship.bigcrunch.com/ads.js",
```

Might be safer in case any platforms interpret the `?` as being for a regex